example/.eslintrc

```diff
@@ -13,6 +13,9 @@
         "expect": true,
         "protractor": true
     },
+    "rules": {
+        "prefer-arrow-callback": 0
+    },
```

Comment:
I'm getting ~100 `error  Unexpected function expression  prefer-arrow-callback` errors with eslint in wildcat. I'm guessing something somewhere has changed. I know for this project we prefer named function callbacks